spec/jobs/reports/quarterly_account_stats_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe Reports::QuarterlyAccountStats do
+  let(:report_date) { Date.new(2020, 1, 1) }
+
+    subject(:report) { described_class.new.tap { |r| r.report_date = report_date } }
+
+  describe '#perform' do
+    # TK
+  end
+
+  describe '#report_body' do
+    # subject(:report_body) { report.report_body }
+
+    let(:proofed_user_now) { create(:user, :proofed) }
+    let(:base_user) { create(:user) }
+    let(:deleted_user) do
+      user = create(:user)
+      DeletedUser.create_from_user(user)
+      # Can we make create_from_user return the new object???
+      DeletedUser.last
```

Comment:
I initially tried adding a `:deleted` trait in the User factory, but `DeletedUser.create_from_user(user)` doesn't return the DeletedUser object, and it felt slightly weird to have something in the User factory that returned a different class.

`DeletedUser.last` is inelegant and should probably at least user `user.id` or something.